src/Microsoft.Health.Fhir.Core/Features/Validation/AttributeValidator.cs

```diff
@@ -41,7 +42,12 @@ public IEnumerable<ValidationFailure> Validate(PropertyValidatorContext context)
                 {
                     foreach (var error in results)
                     {
-                        yield return new ValidationFailure(error.MemberNames?.FirstOrDefault(), error.ErrorMessage);
+                        ITypedElement typedElement = resourceElement.Instance;
+
+                        var fullFhirPath = typedElement.InstanceType;
+                        fullFhirPath += error.MemberNames?.FirstOrDefault() == null ? $"" : "." + error.MemberNames?.FirstOrDefault();
```

Comment:
Small tweak suggestion:
```suggestion
                        fullFhirPath += string.IsNullOrEmpty(error.MemberNames?.FirstOrDefault()) ? string.Empty : "." + error.MemberNames?.FirstOrDefault();
```

Reply:
Ah, right. Thanks. Fixed in dcf6b94.